frontend/src/__tests__/cypress/cypress/pages/modelServing.ts

```diff
@@ -79,8 +79,8 @@ class ModelServingGlobal {
 }
 
 class InferenceServiceModal extends Modal {
-  constructor() {
-    super('Deploy model');
+  constructor(private edit = false) {
+    super(edit ? 'Edit model' : 'Deploy model');
```

Comment:
Small nit(take it or leave it): Can we rewrite this into something like : `super(`${edit ? 'Edit' : 'Deploy'} model`);`